src/client/components/hiluk-menu/hiluk-menu.tsx

```diff
@@ -92,61 +98,60 @@ export class HilukMenu extends React.Component<HilukMenuProps, HilukMenuState> {
     const { openOn, onClose, externalViews, essence, getDownloadableDataset, addEssenceToCollection } = this.props;
     const { url, specificUrl } = this.state;
 
-    var shareOptions: JSX.Element[] = [];
+    const shareOptions: JSX.Element[] = [];
 
     if (addEssenceToCollection) {
       shareOptions.push(<li
         key="add-to-collection"
         className="add-to-collection"
-        onClick={addEssenceToCollection}
-      >{STRINGS.addToCollection}</li>);
+        onClick={addEssenceToCollection}>{STRINGS.addToCollection}</li>);
```

Comment:
why joining these lines? isn't the previous version more readable?

Reply:
in my opinion previous version was much less readable